src/js/angular/graphql/templates/modal/endpoint-generation-failure-result-modal.html

```diff
@@ -0,0 +1,42 @@
+<link href="css/graphql/endpoint-generation-failure-result-modal.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div class="modal-header">
+    <button type="button" class="close" ng-click="close()"></button>
+    <h3 class="modal-title">{{'graphql.create_endpoint.wizard_steps.generate_endpoint.generation_failure_report.title' | translate}}</h3>
+</div>
+
+<div class="modal-body endpoint-generation-failure-result-modal-body">
+    <p>{{'graphql.create_endpoint.wizard_steps.generate_endpoint.generation_failure_report.reason_message' |
+    translate: {endpointId: endpointReport.endpointId} }}</p>
+    <table class="report table table-bordered" aria-describedby="Endpoint generation failure report table">
```

Comment:
Maybe you meant 'aria-label'?

Reply:
yeah, my bad, but I'll change it in my next MR